src/ontology/mondo-edit.obo

```diff
@@ -135083,12 +135084,12 @@ xref: OMIM:108900 {source="MONDO:equivalentTo", source="Orphanet:1479", source="
 xref: Orphanet:1479 {source="MONDO:equivalentTo", source="OMIM:108900", source="DOID:0110112"}
 xref: UMLS:C1862388 {source="MONDO:ncbi_mim2gene_medline", source="MONDO:notFoundInDiseaseSubset"}
 xref: UMLS:C3276096 {source="MONDO:equivalentTo", source="OMIM:108900", source="MONDO:MEDGEN", source="MONDO:notFoundInDiseaseSubset"}
-is_a: MONDO:0005267 {source="MONDO:0015506-obsoleted"} ! heart disorder
 is_a: MONDO:0006664 {source="DC-OMIM:108900", source="DOID:0110112", source="MONDO:Redundant", source="OMIM:108900", source="OMIM:108900/inferred"} ! atrial septal defect
 intersection_of: MONDO:0006664 ! atrial septal defect
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/2488 ! NKX2-5
+relationship: excluded_subClassOf MONDO:0005267 {source="MONDO:0015506-obsoleted", source="https://orcid.org/0000-0001-5208-3432"} ! heart disorder
```

Comment:
'heart disorder' is a great-grandparent. We can remove this parent, but we should not add as an "ecluded_SC"